### PR TITLE
Add message how to create chart on statistics

### PR DESCRIPTION
**Pull request by Daryna-del**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by NatalyMinziuk — APPROVED** [review]

File: src/pages/Statistics/StatisticsCities.tsx [hunk 1]
```diff
@@ -421,33 +421,50 @@ const onClickReset = () => {
...
-            showLessItems: true,
-            responsive: true,
-            showSizeChanger: true,
-          }}
-        />}        
-      </div>
-    </Layout.Content>
+      <Form>
+        <Form.Item>
+          <Row style={{fontSize: '20px'}} gutter={[5, 0]}>
```
Comment (VSBalanchuk) [inline src/pages/Statistics/StatisticsCities.tsx:426]: It is bad practise to use inline styles.
Please change it

File: src/pages/Statistics/StatisticsCities.tsx [hunk 2]
```diff
@@ -421,33 +421,50 @@ const onClickReset = () => {
...
+            <Col>
+              <AntTooltip title="Для того, щоб сформувати діаграму даних станиці клацніть один раз на рядок в таблиці тієї станиці. 
+                                    Діаграму не можливо сформувати, якщо немає даних!">
+                <InfoCircleOutlined/>
+              </AntTooltip>   
+            </Col>
+            <Col>
+              <label style={{fontSize: '15px'}}>Як сформувати діаграму?</label>
```
Comment (VSBalanchuk) [inline src/pages/Statistics/StatisticsCities.tsx:434]: Same here

File: src/pages/Statistics/StatisticsRegions.tsx [hunk 3]
```diff
@@ -414,6 +414,21 @@ const StatisticsCities = () => {
           }
         </div>
         {showTable === false ? "" :
+        <Form>
+          <Form.Item>
+            <Row style={{fontSize: '20px'}} gutter={[5, 0]}>
```
Comment (VSBalanchuk) [inline src/pages/Statistics/StatisticsRegions.tsx:419]: Same here

File: src/pages/Statistics/StatisticsRegions.tsx [hunk 4]
```diff
@@ -414,6 +414,21 @@ const StatisticsCities = () => {
...
+              <Col>
+                <AntTooltip title="Для того, щоб сформувати діаграму даних округи клацніть один раз на рядок в таблиці тієї округи. 
+                                  Діаграму не можливо сформувати, якщо немає даних!">
+                  <InfoCircleOutlined/>
+                </AntTooltip>   
+              </Col>
+              <Col>
+                <label style={{fontSize: '15px'}}>Як сформувати діаграму?</label>
```
[the remark this answers is not in the document]
Comment (Daryna-del) [reply src/pages/Statistics/StatisticsRegions.tsx]: Thanks, changed

File: src/pages/Statistics/StatisticsRegions.tsx (hunk 3 above)
Comment (Daryna-del) [reply src/pages/Statistics/StatisticsRegions.tsx]: Thanks, changed

File: src/pages/Statistics/StatisticsCities.tsx (hunk 2 above)
Comment (Daryna-del) [reply src/pages/Statistics/StatisticsCities.tsx]: Thanks, changed

File: src/pages/Statistics/StatisticsCities.tsx (hunk 1 above)
Comment (Daryna-del) [reply src/pages/Statistics/StatisticsCities.tsx]: Thanks, changed

**Review by VSBalanchuk — APPROVED** [review]

**Review by Bodasyuk — APPROVED** [review]

**Pull request merged**